healthcare/healthcare/custom_doctype/sales_invoice.py

```diff
@@ -0,0 +1,43 @@
+import frappe
+
+from erpnext.accounts.doctype.sales_invoice.sales_invoice import SalesInvoice
+
+
+class HealthcareSalesInvoice(SalesInvoice):
+	@frappe.whitelist()
+	def set_healthcare_services(self, checked_values):
+		self.set("items", [])
```

Comment:
Why do we clear the table? Is there any particular reason?

Reply:
This was added long back. I am not sure why it was added. 

Users can add items multiple times to the sales invoice. Will remove it.

Thanks, @rtdany10 

cc @ruchamahabal

Reply:
Fixed in https://github.com/frappe/healthcare/pull/53